pb_plugins/dcsdkgen/methods.py

```diff
@@ -113,38 +113,38 @@ def collect_methods(
         _methods = {}
         for method in methods:
             # Check if method is completable
-            if (is_completable(method, responses)):
-                _methods[method.name] = CompletableMethod(plugin_name,
-                                                          package,
-                                                          template_env,
-                                                          method,
-                                                          requests,
-                                                          responses)
+            if (no_return(method, responses)):
+                _methods[method.name] = Call(plugin_name,
+                                             package,
+                                             template_env,
+                                             method,
+                                             requests,
+                                             responses)
 
             # Check if method is observable
-            elif (is_observable(method)):
-                _methods[method.name] = ObservableMethod(plugin_name,
-                                                         package,
-                                                         template_env,
-                                                         method,
-                                                         requests,
-                                                         responses)
+            elif (is_stream(method)):
+                _methods[method.name] = Request(plugin_name,
+                                                package,
+                                                template_env,
+                                                method,
+                                                requests,
+                                                responses)
 
             else:
-                _methods[method.name] = SingleMethod(plugin_name,
-                                                     package,
-                                                     template_env,
-                                                     method,
-                                                     requests,
-                                                     responses)
+                _methods[method.name] = Request(plugin_name,
+                                                package,
+                                                template_env,
+                                                method,
+                                                requests,
+                                                responses)
 
         return _methods
 
     def __repr__(self):
         return "method: {}".format(self._name)
 
 
-class CompletableMethod(Method):
+class Call(Method):
     """ A completable method doesn't return any value,
```

Comment:
Comment...

Reply:
I'll run an sed over it, there is so much dronecore still in it :-D